app/controllers/api/v1/code_schools_controller.rb

```diff
@@ -2,7 +2,8 @@ module Api
   module V1
     class CodeSchoolsController < ApplicationController
       def index
-        render json: CodeSchools.all
+        @schools = CodeSchool.all
+        render json: @schools
```

Comment:
Can you pls send me a `gist` of what the output of this would be.

Reply:
https://gist.github.com/PrimeTimeTran/98efd0c629309a62c19c2b06d01c96b7
![screen shot 2017-10-13 at 8 24 13 am](https://user-images.githubusercontent.com/16382165/31526303-231b9c78-aff0-11e7-8012-d08babe64a33.jpg)